src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -1217,6 +1220,18 @@ winrt::TerminalApp::TerminalWindow AppHost::Logic()
     return _windowLogic;
 }
 
+void AppHost::_handleMoveContent(const winrt::Windows::Foundation::IInspectable& /*sender*/,
+                                 winrt::TerminalApp::RequestMoveContentArgs args)
+{
+    _windowManager.RequestMoveContent(args.Window(), args.Content(), args.TabIndex());

```

Comment:
unpack repack unpack repack

Reply:
get ready, we aint even started yet. Layering is silly, but I think it's still valuable to have Remoting _not_ depend on App